scripts/env-init.js

```diff
@@ -24,17 +27,24 @@ const init = async () => {
       })
       .filter(([, value]) => isValue(value)),
   );
+  const content = formatEnvFile({ ...env, ...defaults }, spec);
 
-  await writeFile(
-    './.env',
-    formatEnvFile(
-      {
-        ...env,
-        ...defaults,
-      },
-      spec,
-    ),
-  );
+  try {
+    // Check if file exists before trying to delete
+    if (fs.existsSync(filePath)) {
+      try {
+        fs.unlinkSync(filePath);
+        console.log(`🗑️ Removed existing ${filePath}`);
+      } catch (err) {
+        console.warn(`⚠️ Could not remove existing ${filePath}:`, err.message);
+      }
+    }
+
+    fs.writeFileSync(filePath, content, { encoding: 'utf8' });
+    console.log(`✅ Created new ${filePath} with fresh values.`);
+  } catch (err) {
+    console.error('❌ Failed to write .env file:', err);
+  }
```

Comment:
Waarom is dit nodig? En ik stel voor dat we await en fs/promises gebruiken waar mogelijk